modin/core/io/column_stores/parquet_dispatcher.py

```diff
@@ -15,6 +15,20 @@
 
 import os
 
+import fsspec
+import fsspec.core
+from fsspec.core import split_protocol
+import io
+from fsspec.registry import get_filesystem_class
```

Comment:
let's not mix imports from modules (import stuff from `fsspec` without injecting `import io` between):
```suggestion
import io

import fsspec
import fsspec.core
from fsspec.core import split_protocol
from fsspec.registry import get_filesystem_class
```

also I would like to make imports in 3 different groups separated by an empty line:
1. builtin and stdlib (`os` and `io` in this case)
2. installed as dependency (`fsspec`, `pandas`, etc.)
3. Modin modules (`modin.config`, etc.)